src/main/java/com/ing/data/cassandra/jdbc/CassandraResultSet.java

```diff
@@ -977,8 +977,9 @@ public Object getObject(final int columnIndex) throws SQLException {
                 case ASCII:
                 case TEXT:
                     return this.currentRow.getString(columnIndex - 1);
-                case INT:
                 case VARINT:
+                    return this.currentRow.getObject(columnIndex - 1);
```

Comment:
You're right, it was incorrect to use `getInt` for `varint` type, but to be really correct, it will be even better to use `getBigInteger` instead of `getObject`.

The same fix should be applied at the following locations:
- https://github.com/ing-bank/cassandra-jdbc-wrapper/blob/35e47fecfdde045ae99223df86beef7f6f83075f/src/main/java/com/ing/data/cassandra/jdbc/CassandraResultSet.java#L1117
- https://github.com/ing-bank/cassandra-jdbc-wrapper/blob/35e47fecfdde045ae99223df86beef7f6f83075f/src/main/java/com/ing/data/cassandra/jdbc/CassandraMetadataResultSet.java#L738 (here, we should use `getVarint()`)
- https://github.com/ing-bank/cassandra-jdbc-wrapper/blob/35e47fecfdde045ae99223df86beef7f6f83075f/src/main/java/com/ing/data/cassandra/jdbc/CassandraMetadataResultSet.java#L690 (here, we should use `getVarint()`)